draft-irtf-cfrg-aegis-aead.md

```diff
@@ -1678,7 +1678,7 @@ AEGIS-256 offers 256-bit message security against plaintext and state recovery,
 
 Under the assumption that the secret key is unknown to the attacker, all AEGIS variants offer at least 128-bit security against forgery attacks.
 
-Encrypting the same message with the same key and nonce but different associated data generates distinct ciphertexts that do not reveal any additional information about the message.
+Encrypting the same message with the same key and nonce but different associated data generates distinct ciphertexts that do not reveal any additional information about the message. However, this does not imply that (nonce, key) pairs can be safely reused, even if the associated data differs.
```

Comment:
1. Do we want to go a step further and say SHOULD NOT or MUST NOT intentionally be reused?
2. Elsewhere in the document, we say `(key, nonce) pairs`, so I wonder if they should both be written the same way around.

Reply:
Indeed. MUST NOT would be appropriate here. And good catch for the order of the tuple!

Changed to simply "However, `(key,nonce)` pairs MUST NOT be reused, even if the associated data differs."